zap/src/main/java/org/zaproxy/zap/extension/brk/BreakpointManagementInterface.java

```diff
@@ -30,6 +30,12 @@
 
     boolean isBreakAll();
 
+    boolean isBreakOnJavaScript();
```

Comment:
These are no longer needed?

Reply:
They aren't needed anymore. Should I remover them? What about The API ?

Reply:
You mean the ZAP API? I'd expect the respective implementations to provide endpoints to add/remove the ignored rules.